_docs/plugins/themes.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: documentation
+title: Themes
+category: Plugins
+order: 5.1
+---
+
+# Finding Themes
+
+Themes published on npm should have the keyword `lounge-theme`, so you can find a list by [searching on npmjs.org](https://www.npmjs.com/search?q=keywords%3Alounge-theme).
+
+# Installing Themes
+
+To install theme `lounge-theme-custom`
+
+* `npm install --prefix $LOUNGE_HOME/packages lounge-theme-custom`
```

Comment:
Maybe until `lounge install` is ready, it's worth adding `export $LOUNGE_HOME=<config path of The Lounge>` and `mkdir -p $LOUNGE_HOME/packages` before that line?
See https://github.com/thelounge/lounge/pull/1266#issuecomment-330018302

Reply:
I think we need to set`-g` as well, though I couldn't figure out the differences with/without when using `--prefix`...

Reply:
```sh
$ npm install --prefix $LOUNGE_HOME/packages lounge-theme-solarized
/Users/jeremie/.lounge/packages
└── lounge-theme-solarized@1.0.4

npm WARN enoent ENOENT: no such file or directory, open '/Users/jeremie/.lounge/packages/package.json'
npm WARN packages No description
npm WARN packages No repository field.
npm WARN packages No README data
npm WARN packages No license field.
```

vs.

```sh
$ npm install --global --prefix $LOUNGE_HOME/packages lounge-theme-solarized
/Users/jeremie/.lounge/packages/lib
└── lounge-theme-solarized@1.0.4
```

:man_shrugging:

Reply:
>EDIT: when intalling with --prefix and --global, themes can't be found

That'll be because when installing with `--global`, they get put in `$LOUNGE_HOME/packages/lib/node_modules` rather than `$LOUNGE_HOME/packages/node_modules`.

Definitely possible to do, but we'll need a code change for that obviously.

Happy wither way.

> Maybe until lounge install is ready, it's worth adding export $LOUNGE_HOME=<config path of The Lounge> and mkdir -p $LOUNGE_HOME/packages (EDIT: just setting the env var is enough) before that line?

Hmmm, I was assuming that we were just referring to `$LOUNGE_HOME` in the documentation as where they set their home to be, and they should replace the envar with that. I wasn't actually thinking people would actually use that.  ¯\\\_(ツ)_/¯